tests/defences/detector/poison/test_clustering_analyzer.py

```diff
@@ -18,212 +18,129 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 import logging
-import unittest
 
 import numpy as np
+import pytest
```

Comment:
I think we should decorate all agnostic tests with `@pytest.mark.framework_agnostic` to make sure they run only once.

Reply:
Fixed in commit

Reply:
Fixed in commit